feat: `C-t` can be disabled (#13)
2KAbhishek/tmux-tea: README.md

There are two ways to open tea

- `<prefix> - t`, this can be configured with the `@tea-bind` option e.g: `set -g @tea-bind "t"`

> - <kbd>Ctrl</kbd>+<kbd>t</kbd> alternate binding for a smoother experience. (this can be configured with the `@tea-bind-alt` option and if set to `"false"` this keybind is ignored)

Reviewer: Renaming `tea-bind-alt` to `tea-alt-bind` and tweaking desc

```suggestion
- <kbd>Ctrl</kbd>+<kbd>t</kbd> alternate binding, this can be configured with the `@tea-alt-bind` option, set it to `"false"` to disable
```
